.azure-pipelines/xfel/unix-conda-build.yml

```diff
@@ -100,6 +111,9 @@ steps:
     chmod +x $(Pipeline.Workspace)/modules/xfel_regression/merging_test_data/merge_thermo.csh
     export OMP_NUM_THREADS=4
     libtbx.run_tests_parallel module=uc_metrics module=simtbx module=xfel_regression module=LS49 nproc=4
+    echo "DEBUG"
+    cat mp4k/rank_0*.err
+    echo "DEBUG2"
```

Comment:
These debug code will cause this step to always pass since the last command will always run correctly.

Reply:
We were going to set `OMP_NUM_THREADS` to 2 to get rid of the OpenMP warning and I'm testing a change that enables kokkos on macOS.